webapp/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
```

Comment:
Could you merge this with the `.gitignore` in the root level? I prefer to keep one